src/services/model/conversations.js

```diff
@@ -0,0 +1,36 @@
+import endpoints from 'constants/endpoints';
+import { api } from 'services/api';
+
+const authApi = api.injectEndpoints({
```

Comment:
why is this called authApi?

Reply:
Oh nevermind. This is because of copypasting 😆